llama_stack/distribution/stack.py

```diff
@@ -45,7 +45,7 @@
 from llama_stack.log import get_logger
 from llama_stack.providers.datatypes import Api
 
-logger = get_logger(name=__name__, category="core")
```

Comment:
why this change? there are other places in this PR where you have kept `core` -- I think you should keep it around here and nearby files also.